inbox/util/concurrency.py

```diff
@@ -65,8 +71,17 @@ def wrapped(*args, **kwargs):
 def retry_with_logging(func, logger=None, retry_classes=None,
                        fail_classes=None, account_id=None, provider=None,
                        backoff_delay=BACKOFF_DELAY):
-    def callback():
-        log_uncaught_errors(logger, account_id=account_id, provider=provider)
+    network_errs = [0]
```

Comment:
Hey! So I'm actually not sure either... This stackoverflow question told me this was the only way to reference a value from the parent scope in my callback:

http://stackoverflow.com/questions/7935966/python-overwriting-variables-in-nested-functions

Maybe there's a better way? Definitely weird. I'll add a comment in there.